bedrock/firefox/templates/firefox/new/trailhead/thanks.html

```diff
@@ -20,6 +20,10 @@
   data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
 {% endblock %}
 
+{% block site_header %}
+  {% include 'includes/protocol/navigation/menu-firefox/nav-switch.html' %}
```

Comment:
I'm pretty sure this is causing the integration test failures, becuase there's now a download button on the /thanks page:

<img width="1660" alt="image" src="https://user-images.githubusercontent.com/400117/65583662-89f2be00-df77-11e9-8ec6-d5186311da8f.png">

This button probably shouldn't be here, because clicking it causes a circular look (I think this is what's throwing the test in a loop).

Reply:
Ugh, thank you. Everything was rendering right so I couldn't figure out what the problem was.